moses/tests/neighborhood-sampling-test.metta

```diff
@@ -0,0 +1,262 @@
+!(register-module! ../../../metta-moses)
```

Comment:
Any idea why this file appears to be changed completely? I can't see the changes at all.